docs/index.rst

```diff
@@ -1,23 +1,108 @@
-.. pluggy documentation master file, created by
-   sphinx-quickstart on Mon Nov 14 11:08:31 2016.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+``pluggy``
+==========
 
-Welcome to pluggy's documentation!
-==================================
+The ``pytest`` plugin system
+----------------------------
+``pluggy`` is the crystallized core of `plugin management and hook
+calling`_ for `pytest`_.
 
-Contents:
+In fact, ``pytest`` is itself composed as a set of ``pluggy`` plugins
+which are invoked in sequence according to a well defined set of protocols.
+Some 150+ plugins use ``pluggy`` to extend and customize ``pytest``'s default behaviour.
 
+In essence, ``pluggy`` enables function `hooking`_ so you can build "pluggable" systems.
```

Comment:
I was thinking about a section on *Projects which use `pluggy`* but I didn't know the full list offhand.

@nicoddemus is there any more besides `tox` and `pytest` plugins? `execnet` maybe?